server/tests/security/test_credential_creation.py

```
@@ -0,0 +1,170 @@
+        assert v.rule_id == "persist-crontab"
+
+
+# ---------------------------------------------------------------------------
+# 6. Environment sanitization blocks credential leakage
+# ---------------------------------------------------------------------------
+
+class TestEnvSanitization:
+    """terminal_exec_tool strips secrets from the child-process environment."""
+
+    @staticmethod
+    def _stub_heavy_deps():
```

Ruff (0.15.12)

[warning] Missing return type annotation for staticmethod `_stub_heavy_deps`

(ANN205)